src/main/resources/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink/index.jelly

```diff
@@ -110,15 +116,18 @@
                                     <br/>
                                     ${%folders}: ${folderRole.getFolderNamesCommaSeparated()}
                                     <br/>
-                                    <form action="${rootURL}/${it.urlName}/assignSidToFolderRole" method="POST">
+                                    <form action="${rootURL}/${it.urlName}/assignOrRemoveSidFolderRole" method="POST">
                                         <div class="form-row">
-                                            <label for="assign-sid-folder-${index}" style="margin-right: 10px;">
-                                                ${%assignSid}
+                                            <label for="assign-sid-folder-${folderRole.name}" style="margin-right: 10px;">
+                                                SID
                                             </label>
-                                            <input id="assign-sid-folder-${index}" type="text" name="sid"/>
+                                            <input id="assign-sid-folder-input-${folderRole.name}" type="text" name="sid"/>
                                             <input type="hidden" value="${folderRole.name}" name="roleName"/>
-                                            <input type="submit" style="margin-left: 10px;" value="${%submit}"
-                                                   class="submit"/>
+                                            <br/>
+                                            <input id="assign-sid-folder-submit-${folderRole.name}" type="submit"
+                                                   style="margin-left: 40px;" value="Assign" name="action" class="submit"/>
+                                            <input id="remove-sid-folder-submit-${folderRole.name}" type="submit"
+                                                   style="margin-left: 10px;" value="Remove" name="action" class="submit"/>
```

Comment:
The `value`s of the buttons should work with i18n. So something like `value="${%assign}` would be perfect.